CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 # Contributing guidelines
 
+Want to hack on kubernetes? Yay! We have a [Developer's Guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. And of course if you find something undocumented or incorrect along the way, feel free to send a PR!
```

Comment:
Good idea I'll update

On Friday, July 8, 2016, Euan Kemp <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/kubernetes/kubernetes/pull/28639#discussion_r70155146>
> :
>
> > @@ -1,5 +1,7 @@
> >  # Contributing guidelines
> >
> > +Want to hack on kubernetes? Yay! We have a [Developer's Guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. And of course if you find something undocumented or incorrect along the way, feel free to send a PR!
>
> nit: relative link (docs/devel/README.md).
>
> It also might fit better to add an h2 for this too. "Get hacking" or
> "Developer guide" or such maybe?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28639/files/5e53903502f70024307ed0c154d3a9b1b41d6fd2#r70155146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYNbAVDGeiLCj6ttx4X6s5RnWc0oFJyks5qTt6BgaJpZM4JHinM>
> .
>